packages/pg-v5/commands/backups/capture.js

```diff
@@ -58,8 +52,6 @@ module.exports = {
   needsAuth: true,
   args: [{ name: 'database', optional: true }],
   flags: [
-    { name: 'wait-interval', hasValue: true },
-    { name: 'snapshot' },
```

Comment:
should this actually be removed? or just hidden?

Reply:
@jdxcode based on what was said in the support ticket, I'm under the impression that this was all experimental/legacy code that should be removed.

Reply:
it should be hidden, not necessarily removed unless @heroku/dod-ux isn't going to need it anymore

Reply:
I think the project using `snapshot` is delayed indefinitely and it never had any customer-facing usage; we can safely drop the flag /cc @heroku/dod-infra